x-pack/legacy/plugins/code/server/utils/format_parser.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { set } from 'lodash';
+
+export interface Format {
+  [field: string]: string | Format;
+}
+
+export interface Field {
+  name: string;
+  path: string;
+  format: string;
+}
+
+const BOUNDARY = 'ª––––––––º';
+
+const SPLITTER = '≤∞≥';
+
+export class FormatParser {
```

Comment:
why  I didn't see where this class been used?

Reply:
https://github.com/elastic/kibana/pull/45491/files/5529d9df8ebf1dc1ddd2cef8a39744f549fd8457#diff-ea35085489b2fbbe516eccc84f41cc7cR456